packages/map/src/CdcMap.jsx

```diff
@@ -582,7 +582,8 @@ const CdcMap = ({ className, config, navigationHandler: customNavigationHandler,
           breaks.unshift(0)
         }
 
-        breaks.foreach((item, index) => {
```

Comment:
@tturnerswdev33 - remember to test. Changing this to foreach broke major functionality on maps.